### DISCUSSION TITLE
All platforms works fine but only Android doesn't work

### DISCUSSION THREAD

**Discussion by spikedingo** [WalletConnect/walletconnect-monorepo#3359] [Developer Support] [answered]
We have intergrated WCv2 for serveral months. And most time it works fine with all platforms

but the last version of our apps have some connection issue.

We have apple, android, mac, windows four platforms with our apps.

For the newest version, only Android app can't pair success. It is just pairing, and can never receive the sessionApproval

We checked out the network status, it is Ok, because all other wallet on Android can connect WCv2 successfully, like Rainbow, Onekey, OKwallet

The most weired thing is, our previous version of Android app is not connectable ether ! This version was fully tested and can connect WCv2 very well before.

Can walletconnect tech support team give us some inspirations for us about this problem? 

Thanks a lot !

**Comment by kangfenmao** [comment, marked as answer]
It may be related to this issue.

https://github.com/WalletConnect/walletconnect-monorepo/issues/3235

**Reply by spikedingo** [reply]
Confirmed, it is caused by this issue, great job
